docs/pages/submit/ios.md

```diff
@@ -52,12 +57,19 @@ The command will perform the following steps:
 
 ## Submitting your app using CI
 
-The `eas submit` command is able to perform submissions from a CI environment. All you have to do is ensure that all required information is provided with **eas.json** and environment variables. Mainly, providing the archive source (`--latest`, `--id`, `--path`, or `--url`) is essential. Also, make sure that the iOS Bundle Identifier is present in your [app config file](/workflow/configuration.md).
+The `eas submit` command is able to perform submissions from a CI environment. All you have to do is ensure that all required information is provided with **eas.json** and environment variables.
+
+You must do the following:
+
+- Provide the archive source (`--latest`, `--id`, `--path`, or `--url`).
+- Make sure that the iOS Bundle Identifier is present in your [app config file](/workflow/configuration.md).
+- Set the ASC App ID (`ascAppId` in **eas.json**). The ASC App ID is required to skip the Apple developer log-in process, which will likely not be possible on CI due to the 2FA prompt.
 
-For iOS submissions, you must provide either:
+You'll also need to configure your submission credentials **one** of the following ways:
 
-- `EXPO_APPLE_APP_SPECIFIC_PASSWORD` environment variable along with Apple ID and ASC App ID (`appleId` and `ascAppId` in **eas.json**). The ASC App ID is required to skip the Apple developer log-in process, which will likely not be possible on CI due to the 2FA prompt.
-- Your App Store Connect Api Key with the `ascApiKeyPath`, `ascApiKeyIssuerId`, and `ascApiKeyId` fields set in **eas.json**.
+- Setup your App Store Connect API Key with EAS Servers. You can check the state of your credentials by running `eas credentials` or by running `eas submit -p ios` interactively.
```

Comment:
similar to the recommendation above about `<details><summary>` for non-default flows, we could do that here